packages/hoppscotch-data/src/graphql/index.ts

```diff
@@ -1,3 +1,9 @@
+import { HoppGQLAuth } from "./HoppGQLAuth"
+
+export * from "./HoppGQLAuth"
+
+export const GQLReqSchemaVersion = 2
```

Comment:
Shouldn't this be changed as well? It might create confusion using different casing?
https://github.com/hoppscotch/hoppscotch/blob/ace527e21b56e44a4519dac0186fbf3f064ea86a/packages/hoppscotch-data/src/rest/index.ts#L8

Reply:
yup, that should be changed as well, you can include that in the PR as well if you want.